templates/web/base/main_nav.html

```diff
@@ -6,7 +6,9 @@
 [% BLOCK navitem ~%]
     <li [% liattrs | safe %]>
       [%~ IF c.req.uri.path == uri AND NOT always_url ~%]
-          <span [% attrs | safe %]>[% label %]</span>
+          <span [% attrs | safe %]>[% label %]
+            <span class="screen-reader-only">Current page</span>
```

Comment:
Could it be `(current page)`, think for some reason it's just a bit better in brackets.